src/FSharp.Editing/FSharp.Editing.fsproj

```diff
@@ -39,7 +39,7 @@
     <DocumentationFile>bin\Release\FSharp.Editing.XML</DocumentationFile>
     <OtherFlags>--warnon:1182</OtherFlags>
     <NoWarn>52</NoWarn>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
The warning as error is a bit annoying for unused symbols while developping (due to https://github.com/Microsoft/visualfsharp/issues/1005), maybe we can keep it in Release and remove it from Debug build?

If someone pushes without building in Release (and has warnings) it will still be caught by appveyor / travis.